scripts/tsv_to_mrmatrix.py

```diff
@@ -36,92 +32,122 @@ def coarsen(f, tile_size=256):
         print("curr_size:", curr_size)
         g = r.create_group(str(curr_resolution))
         values = g.require_dataset('values', curr_size, dtype='f4',
-            compression='lzf', fillvalue=np.nan)
+                                   compression='lzf', fillvalue=np.nan)
 
         dask_dset = dask_dset.rechunk((chunk_size, chunk_size))
         dask_dset = da.coarsen(np.nansum, dask_dset, {0: 2, 1: 2})
         da.store(dask_dset, values)
 
 
-def parse(input_handle, output_hdf5, top_n=None):
-    input_handle
-    first_line = next(input_handle)
-    parts = first_line.strip().split('\t')
-    # TODO: Use the python built-in csv module, instead of parsing by hand?
+def parse(input_handle, output_hdf5, height, width,
+          delimiter, first_n, is_square, is_labelled):
+    reader = csv.reader(input_handle, delimiter=delimiter)
+    if is_labelled:
+        first_row = next(reader)
+        labels = first_row[1:(first_n + 1) if first_n else None]
+        if is_square:
+            output_hdf5.create_dataset(
+                'labels',
+                data=np.array(labels, dtype=h5py.special_dtype(vlen=str)),
+                compression='lzf')
+        # TODO: Handle non-square labels
```

Comment:
There should be something warning the user of this lack of functionality, no?

Reply:
Like a print statement to stderr or raising a NotImplementedException

Reply:
@pkerpedjiev : This won't be hard; I'd prefer just to fix it. (Filed as #68.)

Right now I'd prefer to just remove any special handling for symmetric matrices: If there are labels, you'll get `row_labels` and `col_labels` in every case, and if you know that in your case they are equal you can pick one arbitrarily. Does that sound ok?

(Also: Have you run across the [loom](http://loompy.org/) format? I believe it's really just a set of conventions for how to store col and row metadata in hdf5... which is essentially what we want here, with only one piece of metadata, the label. If it adds any complexity, I wouldn't go farther, but if we're just starting and are choosing arbitrary names, that might be a pattern to follow that could save trouble down the road.)

Reply:
> Right now I'd prefer to just remove any special handling for symmetric matrices: If there are labels, you'll get row_labels and col_labels in every case, and if you know that in your case they are equal you can pick one arbitrarily. Does that sound ok?

Yeah, that sounds fine.

Reply:
I'm going to wrap up this PR, and tackle #68 this afternoon.